holonote/annotate/annotator.py

```diff
@@ -343,8 +343,38 @@ def get_display(self, *kdims: str | hv.Dimension) -> AnnotationDisplay:
             self._displays[element_key] = self._create_annotation_element(element_key)
         return self._displays[element_key]
 
-    def __mul__(self, other: hv.Element) -> hv.Overlay:
-        return other * self.get_element(*other.kdims)
+    def _get_kdims_from_other_element(self, other):
+        if isinstance(other, hv.DynamicMap):
+            other = other.last if other.last is not None else other.callback()
```

Comment:
I could imagine that `other.callback()` need some arguments and keyword arguments to work with streams etc.